process/accounts/accountsProcessor.go

```diff
@@ -279,3 +281,11 @@ func (ap *accountsProcessor) computeBalanceAsFloat(balance *big.Int, balancePrec
 
 	return core.MaxFloat64(balanceFloatWithDecimals, 0)
 }
+
+func computeTokenIdentifier(token string, nonce uint64) string {
+	if token == "" || nonce == 0 {
+		return ""
+	}
+
+	return fmt.Sprintf("%s-%d", token, nonce)
```

Comment:
this is wrong. we use `tokenIdentifier-nonce` only for displaying NFTs/SFTs via API. ESDTs should not follow this rule. Also, in order to be consistent with the API pattern, please use a hex encoded nonce